### PR TITLE
[FIX]History Back: Fixed the issue of history back, when calling action ...

### PR DESCRIPTION
**Pull request by msh-odoo**
...history_back it always redirect to home page, instead of previous view

Task: https://www.odoo.com/web#id=11414&view_type=form&model=project.task&menu_id=3942&action=327
Pad: https://pad.odoo.com/p/r.u2AVeY8I323wo3G9

### CODE REVIEW

File: addons/web/static/src/js/chrome.js [hunk 1]
```diff
@@ -706,9 +706,9 @@ instance.web.client_actions.add("reload_context", "instance.web.ReloadContext");
  * If can't go back in history stack, will go back to home.
  */
 instance.web.HistoryBack = function(parent) {
-    if (!parent.history_back()) {
+    $.when(parent.history_back()).fail(function() {
```
Comment (vme-odoo) [inline addons/web/static/src/js/chrome.js]: no need $.when here because history_back method return deffer object in any condition.

Comment (msh-odoo) [inline addons/web/static/src/js/chrome.js]: Done

**Comment by vme-odoo** [comment]
seems good, good job @msh-odoo.

**Comment by ged-odoo** [comment]
merged in 78d4822abbec54f4c97e27fc1921da1b8445453c

Thanks for your work.

**Pull request closed without merging**